source/process/community-guidelines.rst

```diff
@@ -0,0 +1,141 @@
+============================================================
+Mattermost Community Guidelines
+============================================================
+
+In this document we set out some easy-to-follow community guidelines, including:
+
+- `GitHub Help Wanted Issues in Platform Repository`_ - Guidelines for managing Help Wanted issues in GitHub
+- `Community Forums`_ - Guidelines for responding to `Mattermost Forum posts <forum.mattermost.org>`_
+
+----
+
+GitHub Help Wanted Issues in Platform Repository
+---------------------------------------------------------
+
+Goal
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Increase Mattermost usage through more numerous and larger deployments via vibrant open source community by contributing features that would not otherwise be offered
+
+Principles
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- Contributors get a worthwhile benefit from their contributions from the feature directly, and/or from social recognition
+- Help Wanted issues should have a low barrier to contribute
+
+Creating help wanted issues
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Before creating a help wanted issue in the `platform repository <https://github.com/mattermost/platform>`_, confirm the issue you're planning to add has a JIRA ticket associated with it
+
+- JIRA can be used to prioritize help wanted issues internally
+- each issue goes through `Triage meeting <https://docs.mattermost.com/process/training.html#triage-meeting>`_ to ensure the ticket is clear and is approved by at least one developer and product manager
+
+When creating a help wanted issue, please ensure it contains the following (`see a sample help wanted issue here <https://github.com/mattermost/platform/issues/4755>`_):
+
+- is well-defined and unambiguous
```

Comment:
"ensure it contains the following" does make sense with "is well-defined and unambiguous". Maybe either say "Well-defined and unambiguous instructions" or change "please ensure it contain the following" to something that fits with the list